install/bruteforce.sh

```diff
@@ -0,0 +1,3 @@
+sudo apt-get install -y python-pip python-dev
+sudo apt-get install -y libatlas-dev libatlas3gf-base
+sudo apt-get install -y python-numpy
```

Comment:
it seems like the pypi version of numpy gets pulled in from some other package so it might not be needed anyway